src/components/20-molecules/accordion/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@axa-ch/accordeon",
-  "version": "3.0.0",
-  "description": "The accordeon component for the AXA Pattern Library",
+  "name": "@axa-ch/accordion",
+  "version": "1.0.0",
```

Comment:
Ah, I see. You are treating it as a new npm package. That would work ;-)

Can we then mark @axa-ch/accordeon as deprecated on npm? If needed, by publishinhg 3.0.1 with big fat deprecation warning?